analysis_driver/notification.py

```diff
@@ -0,0 +1,24 @@
+from egcg_core import notifications
+
+
```

Comment:
Here, `analysis_driver.NotificationCentre` subclasses `egcg_core.NotificationCentre` to provide boilerplate for starting/ending stages, etc.